pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -329,18 +330,23 @@ func (w *watcher) execute(ctx context.Context, repo git.Repo, repoID string, eve
 		firstRead = false
 	}
 	var (
-		handledEvents    = make([]*pipedservice.ReportEventStatusesRequest_Event, 0, len(eventCfgs))
-		outDatedEvents   = make([]*pipedservice.ReportEventStatusesRequest_Event, 0)
-		maxTimestamp     int64
-		outDatedDuration = time.Hour
-		gitUpdateEvent   = false
+		handledEvents        = make([]*pipedservice.ReportEventStatusesRequest_Event, 0, len(eventCfgs))
+		outDatedEvents       = make([]*pipedservice.ReportEventStatusesRequest_Event, 0)
+		maxTimestamp         int64
+		outDatedDuration     = time.Hour
+		gitUpdateEvent       = false
+		newBranchs           = make([]string, 0, len(eventCfgs))
+		isExistDefaultBranch bool
```

Comment:
We can remove this if we follow https://github.com/pipe-cd/pipecd/pull/4395/files#r1255479042 approach